### PR TITLE
Login Page Completed

### PR DESCRIPTION
**Pull request by MayurChawla**
Created a Login Page

login

### CODE REVIEW

File: Login/Login.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    width: 70%;
+    margin-bottom: 25px;
+}
+.main-section .login-container .password-helper {
+    display: flex;
+    justify-content: space-between;
+}
+.main-section .login-process .btn {
+    border: 2px solid var(--primary-color);
+}
```
Comment (omjadhav85) [inline Login/Login.css:58]: I think .main-section wont be required here...should also work without it. Like below:
.login-container .input-email

File: Login/Login.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    margin: 0;
+    display: block;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 2px solid var(--primary-color);
+    align-items: center;
+    background-color: white;
```
Comment (omjadhav85) [inline Login/Login.css:16]: would suggest to use variables for colors

**Review by omjadhav85 — APPROVED** [review]
Suggested some changes...else code looks good!

File: Login/Login.css (hunk 1 above)
Comment (MayurChawla) [reply Login/Login.css:58]: its a good practise to follow to divide the page in sections and think in terms of components

File: Login/Login.css (hunk 2 above)
Comment (MayurChawla) [reply Login/Login.css:16]: i think var(--white) would internally trigger white, so why not use white directly?

**Pull request merged**